README.md

```diff
@@ -2,31 +2,57 @@
 
 **Tools for validating OPERA RTC products.**
 
+[OPERA RTC Product Information](https://www.jpl.nasa.gov/go/opera/products/rtc-product)
+
+[OPERA Validation Plan](https://d2pn8kiwq2w21t.cloudfront.net/documents/OPERA_Validation_Plan.pdf) 
+
 ---
 ---
 ## Create the `opera_calval_rtc` Conda Environment
 
 If you are working locally:
-  - create the `opera_calval_rtc` conda environment using `environment/environment.yaml`
-    - In a terminal run `mamba env create -f path/to/calval-RTC/environment/environment.yaml`
+  - Make sure you have conda/mamba installed
+    - [Instructions to install miniforge (containing conda and mamba)](https://github.com/conda-forge/miniforge#mambaforge)
+  - Create the `opera_calval_rtc` conda environment using one of the `yaml` files in the `environments` directory
+    - In a terminal, run:
+      - `mamba env create -f path/to/calval-RTC/environment/jlab4_environment.yaml`
+        - Use `calval-RTC/environment/jlab3_environment.yaml` if you'd like to use Jupyter Lab~=3.0 instead of Jupyter Lab~=4.0
+  - Activate your environment and start Jupyter Lab. 
+    - In a terminal run: 
+      - `cd path/to/calval-RTC`
+      - `conda activate opera_calval_rtc`
+      - `jupyter lab`
 
 If you are working in [OpenSARLab](https://opensarlab.asf.alaska.edu/):
   - create the `opera_calval_rtc` conda environment using `environment/osl_environment.yaml`
     - Use the `~/conda_environments/Create_OSL_Conda_Environments.ipynb` notebook to create the environment 
 
+---
+---
+## Start Jupyter Lab in the `opera_calval_rtc` Environment
+
+- In a terminal, run:
+  - `cd path/to/calval-RTC`
+  - `conda activate opera_calval_rtc`
+  - `jupyter lab`
```

Comment:
```suggestion
  - `jupyter lab`
  - `python -m ipykernel install --user --name opera_calval_rtc`
```

I think you need to explicitly install kernel, no?

Reply:
You don't have to, but I think it makes sense to here. If you don't explicitly install the kernel but start jupyter lab from your activated conda environment, you'll end up with the correct kernel, but it will be named `Python3`, which could easily be confusing. Installing the kernel gives you the correct kernel with the expected name.